server/neptune/gateway/event/comment_handler.go

```diff
@@ -91,6 +91,7 @@ func (p *CommentEventWorkerProxy) Handle(ctx context.Context, request *http.Buff
 		// notify user that apply command is deprecated on platform mode
 		if cmd.Name == command.Apply {
 			p.handleLegacyApplyCommand(ctx, event, cmd)
+			return nil
```

Comment:
Would this be equivalent?
```
	if !shouldAllocate {
		return p.forwardToSns(ctx, request)
	}

	if cmd.ForceApply {
		p.logger.InfoContext(ctx, "running force apply command")
		if err := p.vcsClient.CreateComment(event.BaseRepo, event.PullNum, warningMessage, ""); err != nil {
			p.logger.ErrorContext(ctx, err.Error())
		}
		return p.scheduler.Schedule(ctx, func(ctx context.Context) error {
			return p.forceApply(ctx, event)
		})
	}

	if cmd.Name == command.Apply {
		p.handleLegacyApplyCommand(ctx, event, cmd)
	}
	return nil
}
```

Reply:
also: `handleLegacyApplyCommand(ctx context.Context, event Comment, cmd *command.Comment)` remove `cmd` param?